lib/jquery-ui-1-12.js

```diff
@@ -10,20 +9,25 @@ categories = {
 		description: "A required dependency, contains basic functions and initializers.",
 		order: 0
 	},
+	"Core": {
```

Comment:
This needs to be addressed in UI itself, improving the categories we use for "Core" files. Currently we have both "UI Core" and "Core", which makes no sense. We may want to create a "Utilities" category.